### PR TITLE
Disable autocorrect/autocapitalize for the username first letter

### PR DESCRIPTION
**Pull request by ita1024**
Disable autocorrect/autocapitalize for the username first letter in the login form on mobile

### CODE REVIEW

File: app/views/sessions/new.mobile.haml [hunk 1]
```diff
@@ -27,7 +27,7 @@
             .control-group
               = f.label :username, t('username')
               .controls
-                = f.text_field :username, :autofocus => true
+                = f.text_field :username, :autofocus => true, :autocapitalize => 'none', :autocorrect => 'off'
```
Comment (svbergerem) [inline app/views/sessions/new.mobile.haml]: Please use ruby 1.9 hash syntax:
```ruby
= f.text_field :username, autofocus: true, autocapitalize: 'none', autocorrect: 'off'
```
(we [recenlty decided](https://www.loomio.org/d/bLH78pEh/styleguides) to use a [ruby styleguide](https://github.com/diaspora/diaspora/blob/develop/.rubocop.yml))

**Comment by svbergerem** [comment]
Your commit is not linked to your GitHub account and you might want to fix that. For more information see https://help.github.com/articles/setting-your-email-in-git/ and https://help.github.com/articles/changing-author-info/

**Comment by ita1024** [comment]
Apparently the whole file would need to be updated for the hash syntax in question.

**Comment by svbergerem** [comment]
@ita1024 
> Apparently the whole file would need to be updated for the hash syntax in question.

No, it is ok to just use the new hash syntax for the one line you changed. You don't have to change anything else in that file.

**Pull request closed without merging**